synapse/util/file_consumer.py

```diff
@@ -0,0 +1,158 @@
+# -*- coding: utf-8 -*-
+# Copyright 2018 New Vecotr Ltd
```

Comment:
vecotr

Reply:
it's almost as if you're typing this by hand instead of having your editor generate it...